devicepluginmodbuscommander.json

```diff
@@ -0,0 +1,316 @@
+                            "id": "45dfc828-f238-4263-89a3-9b35cf5dea39",
+                            "name": "baudRate",
+                            "displayName": "Baud rate",
+                            "type": "int",
+                            "defaultValue": 9600
+                        },
+                        {
+                            "id": "a27c664b-9f43-4573-a2cc-f65a8fa1a069",
+                            "name": "dataBits",
+                            "displayName": "Data bits",
+                            "type": "int",
+                            "defaultValue": 8
+                        },
+                        {
+                            "id": "4ea8bcdf-d4c5-45a4-a54f-f10ac3f08a78",
+                            "name": "stopBits",
+                            "displayName": "Stop bits",
+                            "type": "int",
+                            "defaultValue": 1
+                        },
+                        {
+                            "id": "72de1b08-2a27-49c5-90e0-8788c3ea1da3",
+                            "name": "parity",
+                            "displayName": "Parity",
+                            "type": "QString",
+                            "inputType": "TextLine",
+                            "allowedValues": [
+                                "No Parity",
+                                "Even Parity",
+                                "Odd Parity"
+                            ],
+                            "defaultValue": "Even Parity"
+                        }
+                    ],
+                    "stateTypes": [
+                        {
+                            "id": "dffc59fe-b230-4345-81d6-0a55f9e16520",
+                            "name": "connected",
+                            "displayName": "connected",
+                            "displayNameEvent": "connection status changed",
+                            "type": "bool",
+                            "defaultValue": false
+                        }
+                    ]
+                },
+                {
+                    "id": "f53524ea-1d06-40a9-b7a4-041297b21e84",
+                    "name": "coil",
+                    "displayName": "Coil",
+                    "createMethods": ["discovery"],
+                    "interfaces": ["connectable"],
+                    "paramTypes": [
+                        {
+                            "id": "ecfae4c3-d8ab-4bea-8bf5-d281740b75f0
+",
+                            "name": "parentId",
+                            "displayName": "Parent Id",
+                            "type": "QString",
+                            "defaultValue": ""
+                        },
+                        {
+                            "id": "d85977a2-4f9c-40f8-9aff-76cea7bd17a3",
+                            "name": "slaveAddress",
+                            "displayName": "slave address",
+                            "type": "int",
```

Comment:
if the address has to be a positiv value, use `uint`

Reply:
also, min/max values possible?

Note that if there is no min/max, a client will add a textfield with number input, if there are, it will use a slider or spinner instead.